### PR TITLE
Move Typescript definitions into our repo

### PR DESCRIPTION
**Pull request by tajo**

### CODE REVIEW

File: scripts/ts-copy.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const fs = require('fs-extra');
+const globby = require('globby');
+const path = require('path');
+
+(async () => {
+  const files = await globby(['src/**/*.d.ts']);
+  files.forEach(async file => {
+    const from = path.join(__dirname, '../', file);
+    const to = path.join(__dirname, '../', file.replace('src', 'dist'));
+    await fs.copy(from, to);
```
Comment (gergelyke) [inline scripts/ts-copy.js]: node now has a copy method, so yo don't need this dep : https://nodejs.org/api/fs.html#fs_fs_copyfilesync_src_dest_flags

File: scripts/ts-copy.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+*/
+
+/* eslint-env node */
+// @flow
+
+const fs = require('fs-extra');
+const globby = require('globby');
+const path = require('path');
+
+(async () => {
```
Comment (gergelyke) [inline scripts/ts-copy.js]: as we use this for build-time only, I think it's safe to use sync apis - I'd do either that, or add a catch to the async function to make sure we see errors if any

File: scripts/ts-copy.js (hunk 1 above)
Comment (tajo) [reply scripts/ts-copy.js]: Do we use Node v12 in our CI?

Comment (tajo) [reply scripts/ts-copy.js]: Oh, it's already in 10

File: scripts/ts-copy.js (hunk 2 above)
Comment (tajo) [reply scripts/ts-copy.js]: It still throws errors

```
(node:35662) UnhandledPromiseRejectionWarning ... specific error message
```

`flow-copy-src` is using the same pattern

**Pull request merged**